src/looper.ts

```diff
@@ -185,8 +190,15 @@ export function idFeeder(
   }
 
   const abort = () => {
-    intervals.forEach((i) => clearInterval(i))
+    for (const i of intervals) clearInterval(i)
   }
 
   return abort
 }
+
+/**
+ * clearProbeInterval clear all probing process
+ */
+export function clearProbeInterval() {
+  clearInterval(probeInterval)
```

Comment:
this doesn't actually clear all intervals because probeInterval is just one interval. to clear all intervals, you should actually use the code in line 193.

Reply:
updated